db/migrate/20201111115249_remove_vacancies_constraints.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveVacanciesConstraints < ActiveRecord::Migration[6.0]
+  def change
+    change_column_null :vacancies, :job_title, true
```

Comment:
When would we want to have a `NULL` job title?

Reply:
When we have to create a vacancy without one 😞 . Designers!!! 😡 😆